stable/mongodb-replicaset/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: mongodb-replicaset
 home: https://github.com/mongodb/mongo
-version: 3.0.0
+version: 3.0.1
```

Comment:
You're adding a feature, please bump the minor version.

Reply:
Hello, thanks for the suggestion. I pushed a commit updating the version to 3.1.0.